src/commands/prepare.ts

```diff
@@ -504,6 +506,15 @@ export async function releaseMain(argv: ReleaseOptions): Promise<any> {
   await switchToDefaultBranch(git, defaultBranch);
 }
 
+/**
+ * Gets Git remote name to use
+ *
+ * @returns Git remote name from environment or default
+ */
+function getRemoteName(): string {
```

Comment:
I was thinking of moving this to `utils/githubApi.ts`, so it can be reused by all the different files.

Reply:
This does `process.env.CRAFT_REMOTE || 'origin'` and in `GithubRemote` we need `name || 'origin'`.
This is looking overcomplicated in my head, would you like to give it a shot with code?

Reply:
The updated patch only touches `prepare.ts`. @tonyo and I decided it is of little value to create a const = 'origin` somewhere just to import it in two places.

Therefore this small func stays here and is used only in this file.